.gitignore

```
@@ -1,2 +1,7 @@
 *__pycache__*
 *.egg-info/
+
+docs/build/
+docs/source/
+!docs/source/index.rst
+!docs/source/CONTRIBUTING.rst
```

```suggestion
!docs/source/CONTRIBUTING.rst
!docs/source/conf.py
!docs/source/_templates/
```

these are also needed as far as I know

---

+1